### PR TITLE
When generating PDF docs, also save it's source as a single-page version of HTML.

### PR DESCRIPTION
**Pull request by nebolsin**

### CODE REVIEW

File: lib/lurker/templates/layouts/print.html.erb [hunk 1]
```diff
@@ -20,6 +20,8 @@
   <meta name="pdfkit-footer-left" content="[title]"/>
   <meta name="pdfkit-footer-center" content="[page] of [toPage]"/>
   <meta name="pdfkit-footer-right" content="[section]"/>
+
+  <link href="<%= @service_presenter.asset_path('application.css') %>" media="all" rel="stylesheet">
```
Comment (Strech) [inline lib/lurker/templates/layouts/print.html.erb]: plz close the tag `/>`

Comment (nebolsin) [inline lib/lurker/templates/layouts/print.html.erb]: In HTML5 these "self-closing" slashes are optional and we actually don't use them in other places (see https://github.com/razum2um/lurker/blob/master/lib/lurker/templates/layouts/application.html.erb#L7). I removed them in meta-tags above for consistency.

Comment (Strech) [inline lib/lurker/templates/layouts/print.html.erb]: :+1:

**Comment by razum2um** [comment]
merged via https://github.com/razum2um/lurker/pull/117

**Pull request closed without merging**